x-pack/plugins/security_solution/public/common/components/alerts_treemap/lib/flatten/flatten_bucket.ts

```diff
@@ -18,6 +19,6 @@ export const flattenBucket = ({
     doc_count: bucket.doc_count,
     key: bucket.key_as_string ?? bucket.key, // prefer key_as_string when available, because it contains a formatted date
     maxRiskSubAggregation: bucket.maxRiskSubAggregation,
-    stackByField1Key: x.key_as_string ?? x.key,
+    stackByField1Key: x.key_as_string ?? (isArray(x.key) ? x.key[0] : x.key),
```

Comment:
Is this a separate treemap fix for array values, or can the new grouping feature somehow impact the treemap?

Reply:
No, it shouldn't impact treemap, we've just extended the `RawBucket` response type to support `string[]` for values (in case of `multi_terms` aggregation) in addition to string.